bluemira/magnetostatics/semianalytic_2d.py

```diff
@@ -339,8 +339,13 @@ def semianalytic_psi(
 
     Notes
     -----
-    Integrates x*Bz to resolve psi. More analytical approaches are possible and
-    will no doubt be faster.
+    The function returns
+
+    .. math::
+        2 \\times 10^{-7} {\\frac{1}{4 d_{xc} d_{zc}}} \\psi
```

Comment:
This is technically right but ignores the full calculation of psi. It would be nice to put that somewhere but probably not for now, could you put a note somewhere in the original issue for this

Reply:
commented in the original issue